modules/slf4j-common/src/main/scala/org/legogroup/woof/slf4j/Slf4jWoofLoggerImpl.scala

```diff
@@ -0,0 +1,108 @@
+package org.legogroup.woof.slf4j
+
+import cats.Functor
+import cats.effect.std.Dispatcher
+import org.legogroup.woof.*
+import cats.syntax.all.*
+
+import scala.util.Try
+
+trait Slf4jWoofLoggerImpl[F[_], Marker](name: String):
+
+  private def getLogInfo() =
+    val stacktraceElements = Thread.currentThread().getStackTrace()
```

Comment:
I know this is not new code, but this could be `(new Throwable).getStackTrace()` for improved performance.
https://stackoverflow.com/a/10625065